src/content/tutorials/fleet-management/cluster-management/aws-cilium-eni-mode/_index.md

```diff
@@ -83,27 +79,38 @@ spec:
   version: # [...]
 ```
 
-Choose a CIDR for the pod network.
-
 Edit the values in the YAML file:
 
-TODO highlight the changed lines (Hugo supports it: https://gohugo.io/content-management/syntax-highlighting/#highlight-shortcode)
+- Set value [`global.connectivity.cilium.ipamMode=eni`](https://github.com/giantswarm/cluster-aws/blob/main/helm/cluster-aws/README.md#connectivity)
+- Set value [`global.connectivity.network.pods.cidrBlocks`](https://github.com/giantswarm/cluster-aws/blob/main/helm/cluster-aws/README.md#connectivity) to the CIDR you want for the pods. This will be associated as secondary CIDR to the VPC.
 
-```yaml
+  In this example, we use the value `10.1.0.0/16`. It must not overlap with the primary VPC CIDR (default `global.connectivity.network.vpcCidr=10.0.0.0/16`).
+- If you chose a different CIDR than `10.1.0.0/16`, please also set [`global.connectivity.eniModePodSubnets`](https://github.com/giantswarm/cluster-aws/blob/main/helm/cluster-aws/README.md#connectivity), for example by copy-pasting the documented default list of subnets and changing the CIDR split (default: `10.1.0.0/16` split into three subnet blocks `10.1.0.0/18`, `10.1.64.0/18`, `10.1.128.0/18`).
+
```

Comment:
maybe we should mention that the POD cidr is added via VPCAdditionalVPCCidr as resource and thus these restrictions apply https://docs.aws.amazon.com/vpc/latest/userguide/vpc-cidr-blocks.html#add-cidr-block-restrictions

IE: if the VPC uses a cidr from  `10.0.0.0/8` customer cannot assign a podCidr a block from `172.16.0.0/12` and `192.168.0.0/16` as that is not allowed will cause AWS to error

Reply:
Added a note with that link, thank you!